server/controllers/comment.controller.js

```diff
@@ -70,6 +67,41 @@ class CommentController {
       return Response(res, 201, 'thread comment added', response);
     }
   }
+
+  /**
+   * Like comment controller
+   * @param {Object} req
+   * @param {Object} res
+   * @returns {Object} response
+   */
+  static async likeComment(req, res) {
+    const { id } = req.user;
+    const { commentid } = req.params;
+
+    const commentResponse = await comments.findOne({
+      attributes: ['comment'],
+      where: { id: commentid },
+    });
+    if (!commentResponse) return Response(res, 404, 'Comment not found');
+    const { comment } = commentResponse.dataValues;
+
+    const likeComment = await commentlikes.create({
```

Comment:
Is there any way of knowing if it is a comment on a draft article?

Reply:
Yes. There may not be a need for a check since we have a separate table for draft comments. I am querying the comment table directly.
No draft has comments in the comment table.